service/controller/chart/v1/resource/release/error.go

```diff
@@ -0,0 +1,12 @@
+package release
```

Comment:
I've renamed the `chart` resource to `release`. I think its a better name since the resource creates Helm releases from chart CRs.